src/server.ts

```diff
@@ -31,22 +33,25 @@ function getLoggerOptions() {
     };
   }
 
-  return process.env.LOGGING === "default";
+  // Don't forget to configure it with
+  // a truthy value in production
+  return !!process.env.LOGGING;
```

Comment:
I generically prefer to expose a `LOG_LEVEL` env variable instead to use in the pretty transport or here.

Reply:
Do you mean something like this?
``ts
return { level: process.env.LOG_LEVEL ?? "info" };
```

Reply:
```
return { level: process.env.LOG_LEVEL || 'silent' };
```

So the application does not log during tests